README.md

```diff
@@ -36,7 +36,9 @@ Alternativly, simply add these lines to your /etc/hosts:
 
 4. `npm run build` (only required when setting up for the first time)
 
-5. `npm run start` to run against stage or prod environments. <br/>
+5. `npm run start:stage` to run against the stage environment <br/>
```

Comment:
Thank you! However, shouldn't it be at least separate commit or maybe even PR? (Mostly interested here what is your git policy :) )

Reply:
I usually just add a separate section to my PR description for "Additional Changes" a separate commit is fine for this. 
I normally wouldn't bother with a separate PR, I find too tight of a commit policy results in me not fixing small bugs I otherwise would due to laziness.

Reply:
thanks @Andrewgdewar, will clarify in the description and add in a separate commit for this!